src/assets/fonts.tsx

```diff
@@ -8,4 +8,26 @@ injectGlobal`
         font-weight: normal;
         font-style: normal;
     }
-`
\ No newline at end of file
+`
+
+const primaryFontFamily = `'ITC Avant Garde Gothic W04', 'AvantGardeGothicITCW01D 731075', 'AvantGardeGothicITCW01Dm', 'Helvetica', 'sans-serif'`,
+
+export const primary = {
+    fontFamily: primaryFontFamily,
+    style: `
+        font-family: ${primaryFontFamily};
+        -webkit-font-smoothing: antialiased;
```

Comment:
Hmm, I think it should, though, so we must be doing something wrong.

> (The CSS rules are automatically vendor prefixed, so you don't have to think about it!)

From https://github.com/styled-components/styled-components/blob/master/README.md#L50